src/Stratis.Bitcoin.Features.Consensus/Interfaces/IPowConsensusValidator.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using NBitcoin;
+using Stratis.Bitcoin.Base.Deployments;
+
+namespace Stratis.Bitcoin.Features.Consensus.Interfaces
+{
+    /// <summary>
+    /// Provides functionality for verifying validity of PoW block.
+    /// </summary>
+    public interface IPowConsensusValidator
+    {
+        /// <summary>Consensus options.</summary>
+        PowConsensusOptions ConsensusOptions { get; }
```

Comment:
Hmmm its a pity you didn't check with me first, ideally this class `IPowConsensusValidator` will be gone at some point when we use the Rules framework.

Reply:
As discussed privately it will not be an issue when replaced with the new rules engine. It is there for testing and can be removed as soon as that is put in place. Does not have a big impact on the newly created miner tests.